src/pages/Main/Main.js

```diff
@@ -19,6 +20,8 @@ const Main = () => {
     <main className="mainContainer">
```

Comment:
```suggestion
    <main className="main">
```
* 최상단 태그는 컴포넌트 이름이랑 같게 수정 해 주세요